lib/widget/button.dart

```diff
@@ -121,7 +122,10 @@ class ButtonState extends WidgetState<Button> {
             color: widget._controller.borderColor ?? defaultShape.side.color,
             width: widget._controller.borderWidth?.toDouble() ?? defaultShape.side.width);
       }
-
+      widget._controller.borderGradient!=null?
+      border=GradientRoundedRectangleBorder(gradient: widget._controller.backgroundGradient??LinearGradient(colors: [Colors.red,Colors.black]), width: widget._controller.borderWidth?.toDouble()??defaultShape.side.width, radius:  widget._controller.borderRadius == null ?
```

Comment:
1. you want borderGradient instead of backgroundGradient
2. Why are you using some random gradient on behalf of the users? This is the only time when you can use !
```
controller.borderGraident != null ? border=controller.borderGraident!
```